core/validators.py

```diff
@@ -0,0 +1,6 @@
+from django.core.validators import RegexValidator
+
+PHONE_REGEX_MESSAGE = 'Phone number must be entered in the format: "+999999999" '\
+                      'Up to 15 numbers allowed'
+
+PHONE_REGEX = RegexValidator(regex=r'^\+?1?\d{9,15}$', message=PHONE_REGEX_MESSAGE)
```

Comment:
I think the best idea is to store the phone numbers in a format that we can easily store, and use however we like, because at the end, stuff as the one described above, is very easy to do on the front end, and storing a phone number like that on the backend, would impede our hability to use it programatically.